packages/xo-web/src/xo-app/vm/tab-snapshots.js

```diff
@@ -86,13 +81,10 @@ const INDIVIDUAL_ACTIONS = [
     handler: snapshot => copyVm(snapshot),
     icon: 'vm-copy',
     label: _('copySnapshot'),
-  },
-  {
-    disabled: (snapshot, { canAdministrate }) => !canAdministrate(snapshot),
-    handler: snapshot => cloneVm(snapshot, false),
-    icon: 'vm-fast-clone',
-    label: _('fastCloneVmLabel'),
-    redirectOnSuccess: snapshot => `/vms/${snapshot}/general`,
+    redirectOnSuccess: (snapshot, { userData }) =>
+      snapshot !== undefined
```

Comment:
In which case is this supposed to be `undefined`?

Reply:
Ok then:
```js
redirectOnSuccess: vm => vm && `/vms/${vm}/general`
```

and in [`SortedTable`](https://github.com/Rajaa-BARHTAOUI/xo-web/blob/master/packages/xo-web/src/common/action-button.js#L89-L96):
```js
if (redirectOnSuccess !== undefined) {
  const to = isFunction(redirectOnSuccess)
    ? redirectOnSuccess(result, handlerParam)
    : redirectOnSuccess
  if (to !== undefined) {
    return this.context.router.push(to)
  }
}
```